docs/conf.py

```diff
@@ -96,91 +84,3 @@
     "logo": "jupyter-logo.svg",
     "logo_text_align": "left",
 }
-
-# Add any paths that contain custom static files (such as style sheets) here,
-# relative to this directory. They are copied after the builtin static files,
-# so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
-
-# Custom sidebar templates, must be a dictionary that maps document names
-# to template names.
-#
-# The default sidebars (for documents that don't match any pattern) are
-# defined by theme itself.  Builtin themes are using these templates by
-# default: ``['localtoc.html', 'relations.html', 'sourcelink.html',
-# 'searchbox.html']``.
-#
-html_sidebars = {
-    "**": [
-        "about.html",
-        "navigation.html",
-        "relations.html",
-        "searchbox.html",
-    ]
-}
-
-# -- Options for HTMLHelp output ------------------------------------------
-
-# Output file base name for HTML help builder.
-htmlhelp_basename = "docker-stacksdoc"
```

Comment:
Note how our .reathedocs.yaml describes "all" formats to be rendered, but we now configure only the `html` output format, and not latex/pdf/htmlhelp etc. I suggest that as part of this PR, you also update the RTD config file to have `formats: []` to opt out of the additional formats like PDF files to be built.

We've done so across most jupyterhub projects now. Sometimes we got build failures, not about the `html` build failing, but for a `pdf` build failing or similar which wasn't very important for us to debug and fix because few use them. I personally have found the PDF builds to be a distraction, because i could end up at such search result via google instead of a more updated easy to read result on an actual website.

https://github.com/jupyter/docker-stacks/blob/cfc62597b072ed7fbf92870d00704363e0653779/.readthedocs.yaml#L23-L24